jep/223/README.adoc

```diff
@@ -218,6 +218,12 @@ For the most current discussions on these points (and more) please refer to link
 
 No existing permission types are being removed at this point, which will allow for existing plugins that make use of them to continue to function.
 Removing code references to the "dangerous" permissions is outside the scope of this JEP.
+Since the granting of the `Jenkins.CONFIGURE` is optional, administrators who find the new permission does not provide value for them can simply continue to grant `Jenkins.ADMINISTER` to users who need overall administrator access (`Jenkins.CONFIGURE` is implied by `Jenkins.ADMINISTER`).
+While we do not anticipate that this change will introduce any regressions, we do acknowledge that some cases may be found after the initial release that may result in confusing or broken behavior for some plugins (ex. a configure link is shown for some plugin, but clicking it results in a permission failure when actually accessing its management link).  
+Because of this, we are committed to the following support policy:
+* For issues raised against Jenkins Core, we will review and fix (or provide justification for not fixing) problems related to this change
+* For issues raised against plugins that are bundled with Jenkins, we will review and fix (or provide justification for not fixing) problems related to this change
+* For issues raised against plugins that are not bundled with Jenkins, it will be up to the maintainers of those plugins to address any found issues
```

Comment:
Would need some clarification. If a serious regression is caused by the change, my expectation would be that the JEP sponsors are on the hook to fix it